.github/workflows/ci.yml

```diff
@@ -6,7 +6,37 @@ on:
   pull_request: {}
 
 jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    strategy:
```

Comment:
I don't think we need a matrix if we are only testing on one node-version, but lemme know if you have other plans here to expand this.

Reply:
But this is minor, so definitely not trying to block approval.

Reply:
You're right matrix isn't necessary for a single version, but I like that all our workflow files are defined the same way, and some of them do use multiple versions